src/components/Card/Card.scss

```diff
@@ -0,0 +1,87 @@
+@import '../src/prepare-styles/colors.scss';
+@import '../src/prepare-styles/reset.scss';
+@import '../src/prepare-styles/typography.scss';
+
+.card_main-container{
+  box-sizing: border-box;
+  padding: 32px;
+  display: flex;
+  flex-direction: column;
+  // height: 439px;
+  // width: 221px;
+  width: min-content;
+  border: 1px solid $grayElements;
+}
+
+.card_container{
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+  gap: 8px;
+
+  &-img{
+    width: 148px;
+  }
+
+  &-title{
+    padding-top: 16px;
```

Comment:
it would be better to use margins for such cases. 

paddings represent distance between **border** of element and it's **content**. margins represent distance between the **whole element** and **other element**. 

it wouldn't make difference in performance or how the website looks, but it would be easier to read. please, check this for the whole project.